### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
Another round of updates. Will play-test on the beta server in the next days.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-2023/pr-test (or any other room on that server)

After merging, a backup will be available at `https://beta.virtualtabletop.io/editor/PR2023-pr-test`.

**Comment by ArnoldSmith86** [comment]
Had a test game between two Android devices without problems.

**Review by 96LawDawg — APPROVED** [review]
I can't remember if you need approval on this.  So just in case, here it is.

**Pull request merged**
